triagebot.toml

```diff
@@ -441,6 +441,14 @@ cc = ["@rust-lang/miri"]
 message = "Some changes occurred in need_type_info.rs"
 cc = ["@lcnr"]
 
+[mentions."compiler/rustc_middle/src/ty/relate.rs"]
+message = "Type relation code was changed"
```

Comment:
idk, should this message be scarier?

Reply:
I think it's fine, I trust us to look at the PR and check even without a clear message